base/sysimg.jl

```diff
@@ -12,8 +12,8 @@ import Base.MainInclude: eval, include
 pushfirst!(Base._included_files, (@__MODULE__, abspath(@__FILE__)))
 
 # set up depot & load paths to be able to find stdlib packages
-Base.init_depot_path()
-Base.init_load_path()
+Base.init_depot_path(; only_bundled=true)
+push!(empty!(LOAD_PATH), "@stdlib")
```

Comment:
Should this be `append_default_depot_path!(DEPOT_PATH; only_bundled=true)`? It seems odd for one of these to now respect the user's JULIA_DEPOT_PATH, but for the other to ignore the user's LOAD_PATH. In theory our sysimg.mk file sets both of those variables correctly for the build already (although I think someone pointed out recently that I missed a place)